spec/avro/dsl/test/optional_record.rb

```diff
@@ -0,0 +1,10 @@
+namespace :test
+
+record :message do
```

Comment:
Are we going to have problems defining `test.message` in both `optional_record.rb` and `optional_array.rb`?

Reply:
No because we recreate the model registry before each test in spec_helper.rb.